### PR TITLE
Update documentation in the ldap package

### PR DESCRIPTION
**Pull request by Nuvindu**
# Description
$subject

### CODE REVIEW

File: ballerina/Module.md [hunk 1]
```diff
@@ -4,135 +4,65 @@ LDAP (Lightweight Directory Access Protocol) is a vendor-neutral software protoc
...
-}
+```bash
+bal run
 ```
 
-#### `searchWithType` API
+## Examples
 
-Returns a list of entries that match the given search parameters.
+The DocuSign eSignature connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/ballerina-platform/module-ballerina-ldap/tree/master/examples).
```
Comment (Bhashinee) [inline ballerina/Module.md]: It should be ldap, right? Not `DocuSign`

File: ballerina/Module.md [hunk 2]
```diff
@@ -4,135 +4,65 @@ LDAP (Lightweight Directory Access Protocol) is a vendor-neutral software protoc
...
-
-public function main() returns error? {
-    ldap:LdapResponse modifyDN = check ldapClient->modifyDN(userDN, "CN=Test User2", true);
-}
+configurable string baseUrl = ?;
+configurable int identityMapCapacity = ?;
+configurable map<anydata> originals = ?;
+configurable map<string> headers = ?;
+
+cregistry:Client schemaRegistryClient = check new ({
```
Comment (Bhashinee) [inline ballerina/Module.md]: It should be ldap client.

File: ballerina/Module.md [hunk 3]
```diff
@@ -4,135 +4,65 @@ LDAP (Lightweight Directory Access Protocol) is a vendor-neutral software protoc
...
 public function main() returns error? {
-    ldap:LdapResponse compare = check ldapClient->compare(userDN, "givenName", "Test User1");
+    string schema = string `
+        {
+            "type": "int",
+            "name" : "value", 
+            "namespace": "data"
+        }`;
+
+    int registerResult = check schemaRegistryClient.register("subject-name", schema);
```
Comment (Bhashinee) [inline ballerina/Module.md]: I think this example is incorrect. What was there is correct right?

File: ballerina/client.bal [hunk 4]
```diff
@@ -70,6 +96,10 @@ public isolated client class Client {
 
     # Determines whether a given entry has a specified attribute value.
     # 
+    # ```ballerina
+    # ldap:LdapResponse compare = check ldapClient->compare(userDN, "givenName", "New User");
```
Comment (Bhashinee) [inline ballerina/client.bal]:
```suggestion
    # boolean compare = check ldapClient->compare(userDN, "givenName", "New User");
```

File: examples/access-directory-server/main.bal [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+    // Delete the employee.
+    ldap:LdapResponse response = check ldapClient->delete("CN=User,dc=mycompany,dc=com");
+    io:println(response);
+}
+
+type Employee record {
+    string[] objectClass;
+    string sn;
+};
```
Comment (Bhashinee) [inline examples/access-directory-server/main.bal]: Isn't it better to move this Employee record to the top of the file?

File: ballerina/Module.md (hunk 2 above)
Comment (Bhashinee) [reply ballerina/Module.md]: Please fix these in the Package.md as well.

File: README.md [hunk 6]
```diff
@@ -11,138 +11,62 @@ LDAP (Lightweight Directory Access Protocol) is a vendor-neutral software protoc
...
-Returns a list of entries that match the given search parameters.
+### Step 2: Instantiate a new connector
 
 ```ballerina
-import ballerina/ldap;
-
-public function main() returns error? {
-    anydata[] value = check ldapClient->searchWithType("DC=ad,DC=com", "(givenName=Test User1)", ldap:SUB);
-}
+ldap:Client ldapClient = check new ({
```
Comment (ayeshLK) [inline README.md]: Shall we define the configurables for `hostName`, `port`, `domainName`, and `password` ?

File: ballerina/Module.md [hunk 7]
```diff
@@ -4,135 +4,59 @@ LDAP (Lightweight Directory Access Protocol) is a vendor-neutral software protoc
...
-Returns a list of entries that match the given search parameters.
+### Step 2: Instantiate a new connector
 
 ```ballerina
-import ballerina/ldap;
-
-public function main() returns error? {
-    anydata[] value = check ldapClient->searchWithType("DC=ad,DC=com", "(givenName=Test User1)", ldap:SUB);
-}
+ldap:Client ldapClient = check new ({
```
Comment (ayeshLK) [inline ballerina/Module.md]: ditto

**Review by ayeshLK — APPROVED** [review]
LGTM

**Pull request merged**